src/Components/PlayPauseButton/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { PlayCircle } from "styled-icons/boxicons-regular/PlayCircle";
+import { PauseCircle } from "styled-icons/boxicons-regular/PauseCircle";
+
+const Button = styled.button`
+  color: ${props => (props.color ? props.color : "#000")};
+  background-color: Transparent;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+`;
+
+const PlayPauseButton = ({ isPlay = true, color, size = 48, ...props }) => {
```

Comment:
I like how you are spreading the rest of the props! Would you give it another name like `rest` or something along those lines. It would be good to make it clear that it doesn't contain all the props but only some.

- [ ] rename `props` to something else.